build-system/pr-check.js

```diff
@@ -361,22 +361,22 @@ function main(argv) {
   }
 
   if (process.env.BUILD_SHARD == "integration_tests") {
-    if (buildTargets.has('RUNTIME')) {
-      command.buildRuntime();
-      command.runIntegrationTests();
```

Comment:
That is a separate call, and happens in runAllCommands(). See https://github.com/ampproject/amphtml/blob/master/build-system/pr-check.js#L270